azurerm/data_source_api_management.go

```diff
@@ -39,21 +39,9 @@ func dataSourceApiManagementService() *schema.Resource {
 				Computed: true,
 			},
 
-			"sku": {
-				Type:     schema.TypeList,
+			"sku_name": {
```

Comment:
This is a breaking change. We should retain sku until 2.0.

Reply:
kk

Reply:
It appears your still removing the old property